src/main/java/org/opensearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -148,6 +148,17 @@ public static void main(String[] args) {
             final GRPCConnectionManager connectionManager =
                     new GRPCConnectionManager(settings.getHttpsEnabled());
             final ClientServers clientServers = createClientServers(connectionManager, appContext);
+
+            // Adds a hook to shut down resources after PA process exits due to some reason.
+            Runtime.getRuntime()
```

Comment:
Can we move this into a separate method? and log after shutdown is complete?

Reply:
Moved to a separate method. Regarding logging, in case we don't see any errors/warn messages in log, we can assume shutdown happened successfully. I personally think explicit logging is not needed. Let me know your thoughts.